src/Regulations/RegulationRegistry.php

```
@@ -0,0 +1,240 @@
+	 *
+	 * @return string|null
+	 */
+	public function currentKey( ?Request $request = null ): ?string
+	{
+		$matched = $this->applicableFor( $request );
+
+		if ( null !== $matched ) {
+			return $matched->key();
+		}
+
+		foreach ( $this->all() as $key => $_ ) {
```

PHPMD (2.15.0)

[warning] Avoid unused local variables such as '$_'. (undefined)

(UnusedLocalVariable)